File: csharp/leet_code/1700.cs
public class Solution {
    public int CountStudents(int[] students, int[] sandwiches) {
        int[] count = new int[2];
        foreach (int student in students) {
            count[student]++;
        }
        foreach (int sandwich in sandwiches) {
            if (count[sandwich] == 0) {
                break;
            }
            count[sandwich]--;
        }
        return count[0] + count[1];
    }
}

File: python/leet_code/1700.py
class Solution:
    def countStudents(self, students: list[int], sandwiches: list[int]) -> int:
        count = [0, 0]
        for student in students:
            count[student] += 1

        for sandwich in sandwiches:
            if count[sandwich] > 0:
                count[sandwich] -= 1
            else:
                break

        return sum(count)
